### ISSUE TITLE
[Web Component]: Alert Box - The Storybook examples should include an alert that the visibility prop can be toggled on and off

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#23792]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Details

Users should be able to test out the `visible` prop using Storybook controls so they can interact with screen readers to get a feel for how the component should behave. This will be beneficial for future accessibility testing as well.

## Acceptance Criteria

- [ ] Storybook is updated with a new story for an alert box that can be shown or hidden

**Comment by k80bowman** [comment]
I looked into this one and it seems there need to be some updates to the Storybook stories for the va-alert component. The close-btn-aria-label, disable-analytics, and visible props do not cause the code to change in storybook and we have no individual stories that show these options.
